usaspending_api/spending_explorer/v2/filters/spending_filter.py

```diff
@@ -68,8 +69,11 @@ def spending_filter(alt_set, queryset, filters, _type):
 
             # recipient
             elif key == "recipient":
-                and_alt_set = alt_set.filter(award__recipient__recipient_name=value)
-                alt_set &= and_alt_set
+                or_alt_set = alt_set.filter(
+                    Q(award__latest_transaction__assistance_data__awardee_or_recipient_legal=value)
```

Comment:
Were these changes in usaspending_api/spending_explorer/v2/filters/spending_filter.py performance tested?

Reply:
not throughly or formally, only by setting up the API to point to DEV DB and playing with the app.